Basic-Video-Capturer-Camera-2-Java/app/src/main/java/com/tokbox/sample/basicvideocapturercamera2/MirrorVideoCapturer.java

```diff
@@ -422,7 +423,6 @@ public synchronized int stopCapture() {
             }
 
             captureSession.close();
-            cameraFrame.close();
```

Comment:
https://tokbox.com/developer/sdks/android/reference/com/opentok/android/BaseVideoCapturer.html#stopCapture-- says that stopCapture() is invoked when capturing frames stop. So, it makes sense to free the camera there.
startCapture() can re-initialize the camera again if capture starts again later on.
Moving this from stopCapture() to destroy() means that the camera will be in use by the capturer until application closes.

Reply:
@josemrecio but are the camera and the camera frame the same thing? Note that we indeed close the capture session  in `stopCapture()`.  Will the camera be in use after this call?

I think the purpose is to avoid an error (a race) when the frame was destroyed before the camera did not finish capturing. Not sure how relevant it is for this code.

Reply:
@josemrecio, please advise.  Thanks 🙏

Reply:
@josemrecio, I'm going to resolve, given lack of a response.  Please un-resolve if needed.  Thanks! 🙏